### PR TITLE
DDP-5924 api docs for admin participants lookup endpoint

### PR DESCRIPTION
**Pull request by yufengwng**
Here is the initial API documentation for the new admin participants-lookup endpoint being added as part of DDP-5924.

### CODE REVIEW

**Comment by yufengwng** [comment]
CC @MocanaAtBroad @bskinner @alextanlan @aglushko-broad

**Comment by evgeniipr** [comment]
@yufengwng looks like `proxy` always should be specified (required)?

**Comment by evgeniipr** [comment]
@yufengwng is it correct that `totalCount` always should be specified (even if found rows count <= MAX_COUNT) ?

**Comment by yufengwng** [comment]
@evgeniipr:

> @yufengwng looks like `proxy` always should be specified (required)?

Good catch! Let's make it nullable!

> @yufengwng is it correct that `totalCount` always should be specified (even if found rows count <= MAX_COUNT) ?

Yeah, I think it should always be specified.

**Comment by evgeniipr** [comment]
@yufengwng is it possible that a participant could have more than 1 proxy?

**Comment by yufengwng** [comment]
@evgeniipr yes, it's possible given our current model. However, so far, we've had only one proxy per participant. I think we'll need to work out more things related to access-control before we're ready to see "one participant multiple proxies" in real usage.

**Review by evgeniipr — DISMISSED** [review]
Looks ok

File: pepper-apis/docs/specification/src/components/schemas.yml [hunk 1]
```diff
@@ -2384,3 +2384,91 @@ Statistics:
...
+    - EXITED_AFTER_ENROLLMENT
+    - CONSENT_SUSPENDED
+    - COMPLETED
+ParticipantsLookupResultItem:
+  type: object
+  required:
+    - guid
+    - hruid
+    - legacyAltPid
+    - legacyShortId
```
Comment (yufengwng) [inline pepper-apis/docs/specification/src/components/schemas.yml:2420]: @evgeniipr @aglushko-broad @MocanaAtBroad Added legacy identifiers.

**Review by evgeniipr — APPROVED** [review]

**Pull request merged**